bff-service/src/server.ts

```diff
@@ -0,0 +1,57 @@
+import express from 'express';
+import {AxiosStatic, Method} from "axios";
+
+require('dotenv').config();
+const axios: AxiosStatic = require('axios').default;
+
+const app = express();
+
+app.set('port', process.env.PORT || 3000);
+app.use(express.json());
+
+app.all('/*', (req, res) => {
+    console.log('originalUrl', req.originalUrl);
+    console.log('method', req.method);
+    console.log('body', req.body);
+
+    const recipient = req.originalUrl.split('/')[1];
+    console.log('recipient', recipient);
+
+    const recipientUrl = process.env[recipient];
+    if (recipientUrl) {
+        const axiosConfig = {
+            url: `${recipientUrl}${req.originalUrl}`,
```

Comment:
Cart service seams to be not working + using originalUrl here is not the best idea as in case product service will have /products/... and /products-smth/.... your code would be not working

Reply:
Please fix at least cart api service